projects/packages/search/src/class-helper.php

```diff
@@ -848,6 +848,7 @@ public static function generate_initial_javascript_state() {
 			'overlayOptions'        => array(
 				'colorTheme'        => get_option( $prefix . 'color_theme', 'light' ),
 				'enableInfScroll'   => get_option( $prefix . 'inf_scroll', '1' ) === '1',
+				'enablePostDate'    => get_option( $prefix . 'show_post_date', is_multisite() ? '1' : '0' ) === '1',
```

Comment:
I noticed that you renamed this option to `show_post_date` here, but didn't follow that change throughout. In other locations we have `enablePostDate` and just `postDate` in app-wrapper/index.jsx and other /components locations.

Is there something about the /components locations that made you choose to leave the `postDate` name instead of changing to `showPostDate`?

Reply:
My naming throughout is attempting to mimic what's already there, so notice the option above, the JS variable name is `enableInfScroll` and the DB option is prefix_inf_scroll

I opted for `show_post_date` rather than just `post_date` in that context to make the DB option simpler to parse.